components/sankey/data-bind.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Data Binding
+page_title: Sankey Diagram Data Binding
+description: Data Binding of the Sankey Diagram for Blazor.
+slug: sankey-data-binding
+tags: telerik,blazor,sankey,diagram,chart,data,binding
+published: True
+position: 3
+---
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+
+# Sankey Data Binding
+
+This article describes the data binding mechanism in the Sankey diagram for Blazor and the supported data source type.
+
+## Data Type and Structure
+
+The Sankey diagram for Blazor requires its `Data` parameter to provide all the data for its nodes, links and labels. The `Data` parameter accepts an object of type [`SankeyData`](/blazor-ui/api/Telerik.Blazor.Components.SankeyData) that contains the following properties:
+
+| Property | Type | Description |
+| --------- | ---- | ----------- |
+| `Links` | `SankeyDataLinks` | A collection of [`SankeyDataLink` objects that describe the links](#link). |
+| `Nodes` | `SankeyDataNodes ` | A collection of [`SankeyDataNode` objects that describe the nodes and their labels](#node). |
```

Comment:
These types are custom, so it's good to clarify that they are actually lists: `SankeyDataLinkss` inherits from `List<SankeyDataLink>` and `SankeyDataNodes` inherits from `List<SankeyDataNode>`